packages/fuels-core/src/types/transaction_builders/script_dry_runner.rs

```diff
@@ -128,7 +136,7 @@ impl<R: DryRunner> ScriptDryRunner<R> {
             Default::default(),
             fake_owner,
             1_000_000_000,
-            Default::default(),
+            *base_asset_id,
```

Comment:
This PR addresses the issue where simulations were failing because there was no base asset that would cover the set tip.
It was discovered because devnet wasn't using the `AssetId::default()` as base asset.

Reply:
Shoot, nice find.